packages/zstandard/test_zstandard.py

```diff
@@ -22,10 +22,10 @@ def zstd_backend(selenium, request):
 
 
 @run_in_pyodide(packages=["zstandard"])
-def test_zstandard_compression_and_decompression(selenium, backend):
```

Comment:
You pass `autouse=True` so you shouldn't need to do anything in the tests. Just remove the backend param and remove the assignment `zstd.backend = zstd_backend`. It is all handled by the fixture.

Reply:
Thanks, that makes sense. Addressed in b7d336f57499183037df7eb76cbd8ba8e2920c74.